commands.js

```diff
@@ -2057,6 +2205,54 @@ var commands = exports.commands = {
 			}
 		}
 		user.rename(targetName, targetToken, targetAuth, connection);
+	},
+
+	a: function (target, room, user) {
+		if (!this.can('rawpacket')) return false;
+		// secret sysop command
+		room.add(target);
+	},
+
+	/*********************************************************
+	 * Help commands
+	 *********************************************************/
+
+	commands: 'help',
+	h: 'help',
+	'?': 'help',
+	help: function (target, room, user) {
+		target = target.toLowerCase();
+
+		// overall
+		if (target === 'help' || target === 'h' || target === '?' || target === 'commands') {
+			this.sendReply("/help OR /h OR /? - Gives you help.");
+		} else if (!target) {
+			this.sendReply("COMMANDS: /nick, /avatar, /rating, /whois, /msg, /reply, /ignore, /away, /back, /timestamps, /highlight");
+			this.sendReply("INFORMATIONAL COMMANDS: /data, /dexsearch, /movesearch, /groups, /opensource, /avatars, /faq, /rules, /intro, /tiers, /othermetas, /learn, /analysis, /calc (replace / with ! to broadcast. Broadcasting requires: + % @ & ~)");
```

Comment:
which ones do you recommend removing? ```/avatars``` and ```/opensource``` seem like obvious suspects, but I'm not sure of the relevancy of many of the others.